content/techniques/queues.md

```diff
@@ -194,6 +194,27 @@ const job = await this.audioQueue.add(
 );
 ```
 
+#### Request-scoped job processors
+
+Whether the processor needs to depend on a request scoped provider or be explicitly registered as a scoped processor, as follows:
+
+```typescript
+@Processor({
+  name: 'audio',
+  scope: Scope.REQUEST,
+})
+```
+
+then you can inject the `JOB_REF` via the constructor.
+
+```typescript
+constructor(@Inject(JOB_REF) jobRef: Job) {
+   console.log(jobRef);
+}
+```
+
+> info **Hint** The `JOB_REF` is imported from `@nestjs/bull`
+
 #### Consumers
```

Comment:
This section should be added after "Consumers" and before "Event listeners" headings.

Reply:
Done 👍